packages/wdio-browserstack-service/src/Percy/Percy-Handler.ts

```diff
@@ -29,7 +29,7 @@ class _PercyHandler {
         private _framework?: string
     ) {
         this._isAppAutomate = isAppAutomate
-        if (!_percyAutoCaptureMode || !['click', 'auto', 'screenshot', 'manual', 'testcase'].includes(_percyAutoCaptureMode as string)) {
+        if (_percyAutoCaptureMode && !_percyAutoCaptureMode || !CAPTURE_MODES.includes(_percyAutoCaptureMode as string)) {
```

Comment:
Won't the first two conditions always evaluate to false?

Reply:
No, its correct, the variable is a string and it is null check, have tested it

Reply:
I mean the first two conditions are `_percyAutoCaptureMode && !_percyAutoCaptureMode`. It's like a & !a which evaluates to false. So is that condition even necessary?

Reply:
It does always evaluate to undefined or false
But it is acting a null check for the next expression ->
CAPTURE_MODES.includes(_percyAutoCaptureMode as string)

Reply:
Nope. It's not. It's evaluating to `(a && b) || c`. How is the above condition guaranteeing null check on a?

Reply:
Fixed